### PR TITLE
Add payment terms list request proposal

### PR DESCRIPTION
**Pull request by luisaterrabrito**

### CODE REVIEW

**Review by steftrenson — APPROVED** [review]

**Review by lowiebenoot — APPROVED** [review]

**Review by wouterds — APPROVED** [review]
👏👏👏

File: src/04-invoicing/payment-terms.apib [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Get a list of available payment terms.
+
++ Request (application/json;charset=utf-8)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `c93ddb52-0af8-47d9-8551-441435be66a7` (string)
```
Comment (jenssegers) [inline src/04-invoicing/payment-terms.apib:12]: Not the biggest fan of reusing the data structure here.

**Review by jenssegers — APPROVED** [review]

**Comment by jdrieghe** [comment]
This can be merged after the next deploy

**Comment by jdrieghe** [comment]
This was deployed to production, merging it.

**Review by jdrieghe — APPROVED** [review]

**Pull request merged**